fusor-ember-cli/tests/unit/models/deployment-test.js

```diff
@@ -10,3 +18,27 @@ test('it exists', function(assert) {
   // var store = this.store();
   assert.ok(!!model);
 });
+
+test('isStarted should be false on init', function(assert){
+  var model = this.subject();
+  assert.equal(model.get('isStarted'), false);
+  assert.equal(model.get('isNotStarted'), true);
+});
+
+test('isStarted should be true if foreman_task_uuid is set to a valid uuid', function(assert){
+  var model = this.subject();
+  Ember.run(function() {
+    model.set('foreman_task_uuid', 'db25a76f-e344-48ba-ac77-f29303586dbe');
```

Comment:
This is an example, where the text has a "hard-coded" value that matches dummy data in our api mocks